repl.js

```diff
@@ -11,19 +11,21 @@ const repl = require('repl');
  */
 const socketPathnames = new Set();
 
-process.once('exit', () => {
+// Clean up REPL sockets and child processes on forced exit.
+process.once('exit', code => {
 	socketPathnames.forEach(s => {
 		try {
 			fs.unlinkSync(s);
 		} catch (e) {}
 	});
+	if (code === 129 || code === 130) {
+		process.exitCode = 0;
+	}
```

Comment:
So, the other lines are just style changes, this if-block is the only real change, am I right?